lxcfs.c

```diff
@@ -82,22 +186,27 @@ static void do_reload(void)
 		goto good;
 
 	dlopen_handle = dlopen("/usr/lib/lxcfs/liblxcfs.so", RTLD_LAZY);
```

Comment:
Yes, did you see no errors with lazy?  That's surprising, though I suppose there's reasonable likelyhood that everything we need was already needed during startup, bc we do a lot of stuff during startup.